pkg/sfu/buffer/buffer.go

```diff
@@ -18,24 +18,15 @@ import (
 	"github.com/livekit/protocol/livekit"
 	"github.com/livekit/protocol/logger"
 
+	"github.com/livekit/livekit-server/pkg/sfu/audio"
+	"github.com/livekit/livekit-server/pkg/sfu/twcc"
 	"github.com/livekit/livekit-server/pkg/utils"
 )
 
 const (
 	ReportDelta = 1e9
 )
 
-type twccMetadata struct {
-	sn          uint16
-	arrivalTime int64
-	marker      bool
-}
-
-type audioLevelMetadata struct {
-	level    uint8
-	duration uint32
-}
-
```

Comment:
I just added these this morning, but the callback thing on every packet has not been sitting well with me for a while. So, bit the bullet and refactored it.